packages/flutter/lib/src/material/material_button.dart

```diff
@@ -92,7 +92,7 @@ class MaterialButton extends StatelessWidget {
     this.materialTapTargetSize,
     this.animationDuration,
     this.minWidth,
-    this.height,
+    this.minHeight,
```

Comment:
This would be a breaking change for anyone that is using the old parameter. We usually deprecate the old name in favor of a new one (rather than just renaming it), and _only then_ if it is worth the effort to change the name.

This may be the type of change that is not worth making, we keep a list of those in https://github.com/flutter/flutter/issues/24722. 

I would recommend reading the [breaking change section](https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes) of the wiki for more guidance. After, if you feel this change is still important to make, and you would like to go through the process laid out in the wiki we can see about moving forward here. :)